src/hazelweb/keyboard/Key.re

```diff
@@ -1,6 +1,110 @@
 module Js = Js_of_ocaml.Js;
 module Dom_html = Js_of_ocaml.Dom_html;
 
+/**
+ * Holding down the option key can result in special characters being typed instead of the intended character
+ * This function tries to return the physical key that was typed in a way that respects alternative keyboard layouts
+ */
+let without_option_key =
```

Comment:
we probably want this to happen only on Mac -- pass in is_mac?

Reply:
Implemented this request in commit [`916730d`](https://github.com/hazelgrove/hazel/pull/523/commits/916730deefe0de97ee714cd363235a2bdba1c7c0).